Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -62,6 +110,16 @@ private bool StartSticking(EntityUid uid, EntityUid user, EntityUid target, Stic
         if (component.Blacklist != null && component.Blacklist.IsValid(target))
             return false;
 
+        //makes sure that neither the target nor the host have entities stuck to them or is stuck to entities
+        if (
+            EntityManager.HasComponent<HasEntityStuckOnComponent>(uid)
+            || EntityManager.HasComponent<HasEntityStuckOnComponent>(target)
+            || EntityManager.TryGetComponent(target , out StickyComponent stuckComp)
+            && stuckComp.StuckTo != null
+            || EntityManager.GetComponent<StickyComponent>(uid).StuckTo != null
+        )
```

Comment:
```suggestion
        if (EntityManager.HasComponent<HasEntityStuckOnComponent>(uid)
            || EntityManager.HasComponent<HasEntityStuckOnComponent>(target)
            || EntityManager.TryGetComponent(target, out StickyComponent stuckComp)
            && stuckComp.StuckTo != null
            || EntityManager.GetComponent<StickyComponent>(uid).StuckTo != null)
```

the last one needs to use tryget